sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -1335,4 +1361,42 @@ export class BlobClient extends StorageClient {
     (response as any).blobDownloadStream = undefined;
     return response;
   }
+
+  private getBlobAndContainerNamesFromUrl(
```

Comment:
Is there new updates for this PR? This function have new version with regex implementation?

Reply:
I'll rebase/merge feature/storage branch into this branch, that should fix this.
We are waiting on Brian and the Arch Board design decisions before moving forward with this PR.